packages/firestore/test/lite/integration.test.ts

```diff
@@ -2046,173 +2041,167 @@ describe('withConverter() support', () => {
 });
 
 describe('countQuery()', () => {
-  it('AggregateQuery and AggregateQuerySnapshot inherits the original query', () => {
+  it('AggregateQuerySnapshot inherits the original query', () => {
     return withTestCollection(async coll => {
       const query_ = query(coll);
-      const countQuery_ = countQuery(query_);
-      const snapshot = await getAggregate(countQuery_);
-      expect(countQuery_.query).to.equal(query_);
-      expect(snapshot.query).to.equal(countQuery_);
-      expect(snapshot.query.query).to.equal(query_);
+      const snapshot = await getCount(query_);
+      expect(snapshot.query).to.equal(query_);
     });
   });
 
-  it('empty test collection count', () => {
-    return withTestCollection(async coll => {
-      const countQuery_ = countQuery(coll);
-      const snapshot = await getAggregate(countQuery_);
-      expect(snapshot.getCount()).to.equal(0);
+  it('run count query on empty test collection', () => {
+    return withTestCollection(async collection => {
+      const snapshot = await getCount(query(collection));
+      expect(snapshot.data().count).to.equal(0);
     });
   });
 
-  it('test collection count with 3 docs', () => {
+  it('run count query on test collection with 3 docs', () => {
     const testDocs = [
       { author: 'authorA', title: 'titleA' },
       { author: 'authorA', title: 'titleB' },
       { author: 'authorB', title: 'titleC' }
     ];
-    return withTestCollectionAndInitialData(testDocs, async coll => {
-      const countQuery_ = countQuery(coll);
-      const snapshot = await getAggregate(countQuery_);
-      expect(snapshot.getCount()).to.equal(3);
+    return withTestCollectionAndInitialData(testDocs, async collection => {
+      const snapshot = await getCount(query(collection));
```

Comment:
nit: `query(collection)` -> `collection`.

You can just call `getCount(collection)` when you're not applying any filters on the collection. Please make this change throughout this file.